internal/app/fhir/r5/CodeSystem.go

```diff
@@ -3,27 +3,26 @@ package r5
 import (
 	"fmt"
 
+	"github.com/CDCgov/phinvads-go/internal/database/models/xo"
 	"github.com/google/fhir/go/proto/google/fhir/proto/r5/core/codes_go_proto"
 	datatypespb "github.com/google/fhir/go/proto/google/fhir/proto/r5/core/datatypes_go_proto"
 	r5pb "github.com/google/fhir/go/proto/google/fhir/proto/r5/core/resources/code_system_go_proto"
-	"github.com/CDCgov/phinvads-go/internal/database/models/xo"
 )
 
-func SerializeCodeSystemToFhir(cs *xo.CodeSystem) (*r5pb.CodeSystem, error) {
+func SerializeCodeSystemToFhir(cs *xo.CodeSystem, conceptCount int, concepts []*xo.CodeSystemConcept) (*r5pb.CodeSystem, error) {
 	fhirCS := &r5pb.CodeSystem{
 		Id:           newId(cs.Oid),
-		Status:       &r5pb.CodeSystem_StatusCode{Value: codes_go_proto.PublicationStatusCode_DRAFT},
+		Status:       &r5pb.CodeSystem_StatusCode{Value: codes_go_proto.PublicationStatusCode_ACTIVE},
 		Version:      newString(cs.Version),
-		Name:         newString(cs.Name),
+		Name:         newString(cs.Codesystemcode),
 		Description:  newNullableMarkdown(cs.Definitiontext),
 		Experimental: newBoolean(cs.Legacyflag),
-		Url:          newNullableUri(cs.Sourceurl),
+		Url:          newUri(fmt.Sprintf("https://phinvads.cdc.gov/r5/CodeSystem/%s", cs.Oid)),
 		Date:         newDateTime(cs.Statusdate),
 		Publisher:    newNullableString(cs.Distributionsourceversionname),
-		Title:        newNullableString(cs.Assigningauthorityversionname),
+		Title:        newString(cs.Name),
 		Content:      &r5pb.CodeSystem_ContentCode{Value: 4},
```

Comment:
Maybe we can swap this one too? 🙈 
```suggestion
		Content:      &r5pb.CodeSystem_ContentCode{Value: codes_go_proto.CodeSystemContentModeCode_COMPLETE},
```